index.html

```diff
@@ -29,15 +29,14 @@
 
       <main>
         <h1>Welcome to Matthew Burgan's CSD120 Website</h1>
-        <a href="images/selfie.jpg" target="_blank">
+        <a href="images/selfie.png" target="_blank"><!-- selfie.png > 900KB -->
```

Comment:
jpg is definitely the best choice for a selfie, but the file you uploaded was a png
It's also >900KB which is too large.  Try to keep web images under 500KB